internal/c/parts/gui/gui.cpp

```diff
@@ -433,57 +368,3 @@ bool gui_alert(const char *fmt, ...) {
 
     return true;
 }
-
-#ifndef QB64_WINDOWS
-
-#    define IDOK 1
-#    define IDCANCEL 2
-#    define IDABORT 3
-#    define IDRETRY 4
-#    define IDIGNORE 5
-#    define IDYES 6
-#    define IDNO 7
-#    define MB_OK 0x00000000L
-#    define MB_OKCANCEL 0x00000001L
-#    define MB_ABORTRETRYIGNORE 0x00000002L
-#    define MB_YESNOCANCEL 0x00000003L
-#    define MB_YESNO 0x00000004L
-#    define MB_RETRYCANCEL 0x00000005L
-#    define MB_SYSTEMMODAL 0x00001000L
-
-// This exists because InForm calls it directly.
-// It only supports the "MB_OK" and "MB_YESNO" options
-int MessageBox(int ignore, char *message, char *title, int type) {
```

Comment:
Are we tossing this?

Reply:
Yeah. This isn't going to be used anymore as InForm will not work with 3.7 anyway, because of the font changes.
I've already swapped out the calls for this with _MESSAGEBOX in the PE version of InFrom I'm working on.